### PR TITLE
deprecate libatk and at-spi-atk

### PR DESCRIPTION
**Pull request by ZachBacon**
at-spi2 now includes what these provided.

### CODE REVIEW

File: repo_data/distribution.xml [hunk 1]
```diff
@@ -187,7 +187,20 @@
 		<Package>pidgin-lwqq</Package>
 		<Package>autotrace</Package>
 		<Package>autotrace-devel</Package>
-		<Package>gnome-tweak-tool</Package>
+		<Package>gnome-tweak-tool</Package>		
+		?
```
Comment (joebonrichie) [inline repo_data/distribution.xml:191]: ?

File: repo_data/distribution.xml.in [hunk 2]
```diff
@@ -290,7 +290,21 @@
 		<Package>autotrace-devel</Package>
 
 		<!-- Replaced by gnome-tweaks //-->
-		<Package>gnome-tweak-tool</Package>
+		<Package>gnome-tweak-tool</Package>		
+		
+		<!-- Replaced by at-spi2 //-->?
```
Comment (joebonrichie) [inline repo_data/distribution.xml.in:295]: ?

**Review by joebonrichie — CHANGES_REQUESTED** [review]
Some extra ??s

File: repo_data/distribution.xml (hunk 1 above)
Comment (ZachBacon) [reply repo_data/distribution.xml:191]: Keyboard error I think?

**Pull request merged**